.gitignore

```diff
@@ -1,2 +1,7 @@
-node_modules
-bower_components
+/package/*
+!/package/.gitkeep
+/node_modules
+/es5/**/*.js
```

Comment:
Why were you previously keeping your transpiler results stored in git? The only thing I could think is that it makes it easier to do development by pointing npm at this git repo. In theory doing compilation in the prepack script should make that work -- it's meant to happen after devDependencies are installed, but there's a bug out on that https://github.com/npm/npm/issues/19564. The bug has a workaround specified though, so I'll try it.